creator-node/src/services/stateMachineManager/stateMonitoring/fetchCNodeEndpointToSpIdMap.jobProcessor.ts

```diff
@@ -40,3 +42,26 @@ module.exports = async function ({
     errorMsg
   }
 }
+
+module.exports = async (
+  params: DecoratedJobParams<FetchCNodeEndpointToSpIdMapJobParams>
+) => {
+  const { parentSpanContext } = params
+  const jobProcessor = instrumentTracing({
+    name: 'fetchCNodeEndpointToSpIdMap.jobProcessor',
+    fn: fetchCNodeEndpointToSpIdMap,
+    options: {
+      links: parentSpanContext
+        ? [
+            {
+              context: parentSpanContext
+            }
+          ]
+        : [],
+      attributes: {
+        [tracing.CODE_FILEPATH]: __filename
+      }
+    }
+  })
+  return await jobProcessor(params)
```

Comment:
is it correct to await the jobProcessor here? we're just returning the function right

Reply:
It is correct - we aren't returning the function here